versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     helm = {
       source  = "hashicorp/helm"
-      version = "2.9.0"
+      version = "~> 2.9"
```

Comment:
according to terraform doc https://developer.hashicorp.com/terraform/language/expressions/version-constraints#version-constraint-syntax 
~>  allows only the right most number to increase. 
We don't have strict version constraints for helm sofar, we can use >= 2.9.0